hack/local-up-volcano.sh

```diff
@@ -44,6 +45,7 @@ function install-volcano {
 }
 
 function uninstall-volcano {
+  kubectl delete job volcano-admission-init -n $VOLCANO_NAMESPACE
```

Comment:
Is this action aimed to prevent that reinstall and bring confliction?

Reply:
yes, always got this problem when reinstall.